src/index.html

```diff
@@ -17,16 +17,25 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="form-field">
+      <form data-qa="big">
+        <input
+          type="text"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+          class="search-bar search-bar--size_big"
+        />
+      </form>
+    </div>
+    <div class="form-field">
+      <form data-qa="small">
```

Comment:
The `data-qa` attribute for the form should be `small-search-form` instead of `small` according to the task requirements.